.github/workflows/build-wheels-publish.yml

```diff
@@ -264,6 +272,14 @@ jobs:
           name: sdist-archive
           path: bindings/python/dist/*.tar.gz
```

Comment:
It should be enough to add the following line:

```python3 -m pip install dist/*.tar.gz```

instead of the section added below